packages/scene-composer/src/augmentations/components/three-fiber/viewpoint/ViewCursorWidget.tsx

```diff
@@ -1,23 +1,26 @@
 import React, { useCallback, useContext, useEffect, useMemo, useRef } from 'react';
-import { useFrame, useThree } from '@react-three/fiber';
-import { Mesh as THREEMesh, Object3D as THREEObject3D, Vector3 as THREEVector3 } from 'three';
+import { useFrame, useLoader, useThree } from '@react-three/fiber';
+import { Mesh as THREEMesh, Object3D as THREEObject3D, Vector3 as THREEVector3, Vector3 } from 'three';
+import { SVGLoader } from 'three-stdlib';
+import { TilesGroup } from '3d-tiles-renderer';
 
-import { convertSvgToMesh } from '../../../../utils/svgUtils';
+import { convertSvgToMesh, getDataUri } from '../../../../utils/svgUtils';
 import { getIntersectionTransform } from '../../../../utils/raycastUtils';
 import { sceneComposerIdContext } from '../../../../common/sceneComposerIdContext';
 import { useEditorState } from '../../../../store';
 import { ViewCursorEditSvgString, ViewCursorMoveSvgString } from '../../../../assets/svgs';
 
+export const INIT_SVG_SCALE = 0.003;
```

Comment:
is this going to be huge for smaller scene?

Reply:
No it's always around the same size. I added a video in the description to show that.